chargeRateStructs.cpp

```diff
@@ -0,0 +1,24 @@
+#include "chargeRateStructs.h"
+
+const char* delimiter = ",";
+const int conversionBase = 10; 
+std::list<long> sessionValues = {};
```

Comment:
Does `sessionValues` really need to be global?
It is returned from one function, so can be passed to another? Alternately, it can be passed by reference by the caller?
Globals have a tendency to 'remember' the past even when you don't need it - a source of error to be avoided when possible

Reply:
Got it, thanks Sudeep, removed both global lists, leaving allocation of said object to the mechanism invoking the functions on the latest commit.